androidbrowserhelper/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.6.2
+
+* [#520](https://github.com/GoogleChrome/android-browser-helper/pull/520) Skip animation as
+default. This alters Trusted Web Activity (TWA) behavior, allowing it to launch without waiting
+for the skip animation to complete. This behavior can be reverted by add following lines in twa
```

Comment:
> twa

please consistently use capitalization for abbreviations

Reply:
Yes, changed.